frontend/package.json

```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "URIjs": "^1.14.1",
+    "angular-elastic": "^2.5.0",
```

Comment:
Hmm... when I use `npm shrinkwrap` myself, this updates all sorts of packages. Probably not what I mean to do, do you know how that crazy new stuff works?